src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl

```diff
@@ -55,22 +55,22 @@ default_free_surface(grid; gravitational_acceleration=g_Earth) =
 
 """
     HydrostaticFreeSurfaceModel(; grid,
-                                             clock = Clock{eltype(grid)}(time = 0),
-                                momentum_advection = CenteredSecondOrder(),
-                                  tracer_advection = CenteredSecondOrder(),
-                                          buoyancy = SeawaterBuoyancy(eltype(grid)),
-                                          coriolis = nothing,
-                                      free_surface = default_free_surface(grid, gravitational_acceleration=g_Earth),
-                               forcing::NamedTuple = NamedTuple(),
-                                           closure = nothing,
-                   boundary_conditions::NamedTuple = NamedTuple(),
-                                           tracers = (:T, :S),
-                     particles::ParticlesOrNothing = nothing,
-             biogeochemistry::AbstractBGCOrNothing = nothing,
-                                        velocities = nothing,
-                                          pressure = nothing,
-                                diffusivity_fields = nothing,
-                                  auxiliary_fields = NamedTuple(),
+                                clock = Clock{eltype(grid)}(time = 0),
+                   momentum_advection = default_momentum_advection(grid),
```

Comment:
the docstring and the construction don't seem to match.. am I right?

Reply:
agree